### DISCUSSION TITLE
Plans and roadmap

### DISCUSSION THREAD

**Discussion by AB1908** [st3v3nmw/obsidian-spaced-repetition#512] [General]
Hi @st3v3nmw. I've been rewriting the plugin in React as a newbie project and was curious if you were interested in the idea. I've made a significant amount of progress and overall, I feel like the state and view definitions are much easier to follow at the cost of plugin size. Is this something you're willing to consider? If not, that's totally fine as I'll be maintaining my own fork anyway. 

My use case was that I wanted to spin this into a React Native app down the line as I quite dislike using Obsidian on mobile. The plugin itself is amazing but I find the app itself annoying to use.

Happy to discuss further if you have the time!

https://github.com/AB1908/obsidian-spaced-repetition/tree/testing

**Comment by st3v3nmw** [comment]
Hey @AB1908,
I've personally never used React but I'll look at your fork and see..
Thanks for the idea!
